packages/grafana-ui/tsconfig.json

```diff
@@ -4,6 +4,7 @@
   "exclude": ["dist", "node_modules"],
   "compilerOptions": {
     "rootDirs": [".", "stories"],
+    "noUnusedLocals": false,
```

Comment:
This   should belong to storybook's tsconfig only, as otherwise it influences all grafana-ui code